jobs/jobs/schemas.py

```diff
@@ -343,3 +346,27 @@ class JobProgress(BaseModel):
     finished: int = Field(..., example=1)
     total: int = Field(..., example=1)
     mode: str = Field(..., example="Automatic")
+
+
+class PipelineEngine(BaseModel):
+    name: str
+    resource: str
+    enabled: bool
+
+
+class PipelineEngineSupport(BaseModel):
+    data: List[PipelineEngine]
+
+
+class Pipeline(BaseModel):
+    id: str
+    name: str
+    version: int = Field(default=0)
+    type: str = Field(default="airflow")
```

Comment:
You could do this
```suggestion
    version: int = 0
    type: str = "airflow"
```

Reply:
For me `Explicit is better than implicit`